test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -36,6 +36,8 @@ git clone -b ${SMDEBUG_RELEASE_TAG} https://github.com/awslabs/sagemaker-debugge
 
 echo "Changing to sagemaker-debugger."
 cd sagemaker-debugger
+git checkout save_inputs_asserts # temporary branch change
```

Comment:
Is this temporary as in it will be removed before this PR is merged, or is it temporary as in it will be needed until a few patch versions later in smdebug 0.9.x?

Reply:
This will be removed with a subsequent PR post the DLC release.
We need this change for forwards compatibility.

This branch leverages tests that are only compatible with the latest aws tf binary.